README.rst

```diff
@@ -88,7 +89,10 @@ Before running tests, please ensure that following environment variables are set
     ==> BASIC_AUTH_PASSWORD
     ==> USER_LOGIN_EMAIL
     ==> USER_LOGIN_PASSWORD
-
+    ==> COURSE_RUN
+    ==> COURSE_DISPLAY_NAME
+    ==> COURSE_NUMBER
+    ==> COURSE_ORG
 
 To run all the tests:
 
```

Comment:
As for the other question: For now, I think the documentation should reflect exactly how we run tests on Jenkins. Once we have our paver targets in order AND update the jenkins scripts, we can update the README to reflect this. For the time being, I think we should just have:
* How to run all the e2e tests -> paver e2e_test --exclude=whitelabel
* How to run a specific e2e test -> paver e2e_test (+ nose specifier)
* How to run wl/microsites tests -> paver e2e_wl_test (+ nose specifiers)

The rationale for splitting them for the time being, is that they do not consume the same environment variables. We want people who have never used this repo to be able to easily test/run code the way it will be run on our CI systems. @raeeschachar  does this make sense to you?